### ISSUE TITLE
File size is correct? 

### ISSUE THREAD

**Issue by alexliradesign** [JaekSooley/ChdManMenu2#1]
![screenshot](https://github.com/user-attachments/assets/34578697-1dc9-42e5-b625-d11ded46922c)

Sorry if I missed something buddy, but 60 megs for a console app doesn't make sense. Thanks anyway...

**Comment by JaekSooley** [comment]
Yeah that checks out. It's a 'self-contained' win-x86 application published as a single file, so it contains everything required for the application to run (rather than requiring the .NET framework to be installed on the system).

This was just a personal project I was playing around with that needed to run on multiple systems, so I went with portability and didn't worry about the size.
